eXo/Sources/Controllers/HomePage/HomePageViewController.swift

```diff
@@ -317,6 +317,7 @@ class HomePageViewController: eXoWebBaseController, WKNavigationDelegate, WKUIDe
         configuration.allowsAirPlayForMediaPlayback = true
         configuration.allowsPictureInPictureMediaPlayback = true
         configuration.mediaTypesRequiringUserActionForPlayback = []
+        configuration.applicationNameForUserAgent = "Version/15.0 Mobile/15E148 Safari/604.1"
```

Comment:
Could we use the agent used by the original browser ?

Reply:
The problem in the Safari version (Version/8.0.2 Safari/600.2.5) now deprecated, we should use the new one instead